db/src/main/kotlin/edu/artic/db/Constants.kt

```diff
@@ -0,0 +1,7 @@
+package edu.artic.db
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+
+const val INVALID_FLOOR: Int = Int.MIN_VALUE
```

Comment:
oh wait please add doc here on what this is. `Constants.kt` is a little too vague.

Reply:
Added.